src/Domain.Linnapps/Reports/SaHoldStoryService.cs

```diff
@@ -31,12 +31,45 @@ public ResultsModel GetSaHoldStoriesReportForSalesArticle(string articleNumber)
             {
                 var row = results.AddRow(story.HoldStoryId.ToString().Replace("/", "%2F"));
                 results.SetGridTextValue(row.RowIndex, 0, story.DateStarted.ToShortDateString());
-                results.SetGridTextValue(row.RowIndex, 1, story.DateFinished != null ? ((DateTime)story.DateFinished).ToShortDateString() : story.DateFinished.ToString());
+                results.SetGridTextValue(
+                    row.RowIndex,
+                    1,
+                    story.DateFinished != null
+                        ? ((DateTime)story.DateFinished).ToShortDateString()
```

Comment:
toString on null? Should one of these just be null?

Reply:
I think the second one (in the dateFinished is null case) should just be string.Empty.